index.html

```diff
@@ -63,95 +65,52 @@ <h5 tabindex="0">
     </ul>
   </section>
   <section class="work" tabindex="0" id="work">
-    <div class="card" tabindex="0">
-      <img src="./pictures/SnapshootPortfolio.png" tabindex="0" alt="Project_Image" >
-      <div class="text_side" tabindex="0">
-              <h2 tabindex="0">Tonic</h2>
-              <div class="title">
-                <h5 tabindex="0">CANOPY</h5>
-                <ul class="title_list">
-                  <li tabindex="0">Back End Dev</li>
-                  <li tabindex="0">2015</li>
-                </ul>
-              </div>
-              <p tabindex="0">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-              <ul class="tech_list">
-                <li tabindex="0">html</li>
-                <li tabindex="0">css</li>
-                <li tabindex="0">javaScript</li>
-              </ul>
-              <button class="btn_details">See Project</button>
-      </div>
-    </div>
-        <div class="card_reverse" tabindex="0">
-          <img class="first_reverse" tabindex="0" src="./pictures/SnapshootPortfolio4.png" alt="Project_Image">
-          <div tabindex="0" class="second_reverse text_side">
-            <h2 tabindex="0">Multi-Post<br>
-              Stories</h2>
+  </section>
+  <div class="bottom_section">
+      <section class="about_section" tabindex="0" id="about">
+      <!-- <div id="myModal" class="modal">
+        <div class="modal-content">
+                <span class="closeModal" id="closeModal">&times;</span>
+          <div class="text_side modal_head" tabindex="0">
+            <h2 tabindex="0">Tonic</h2>
```

Comment:
- [ ] Kindly remove the commented code from your project has it helps you have a cleaner codebase